components/InfoCards/DescriptionCard.vue

```diff
@@ -4,8 +4,8 @@
     :headerString="headerString"
     :subheaderString="subheaderString"
     :subheaderURL="subheaderURL"
-    subheaderOn
-    headerOn>
+    :subheaderOn="subheaderOn"
+    :headerOn="headerOn">
```

Comment:
I also do this in some areas of my code, not worth dealing with now, but I feel like there's a shorter way in Vue to pass variables through when they have the same name as the propName. No changes needed here, just noting

Reply:
I made a note to myself to go back and have those also turn off or on if we don't pass any content to the header/subheader